pkg/collector/corechecks/cluster/orchestrator/processors/k8s/pod.go

```diff
@@ -164,6 +164,19 @@ func (h *PodHandlers) ResourceVersion(ctx processors.ProcessorContext, resource,
 	return resourceModel.(*model.Pod).Metadata.ResourceVersion
 }
 
+// ResourceTaggerTags is a handler called to retrieve tags for a resource from the tagger.
+//
+//nolint:revive // TODO(CAPP) Fix revive linter
+func (h *PodHandlers) ResourceTaggerTags(ctx processors.ProcessorContext, resource interface{}) []string {
+	r := resource.(*corev1.Pod)
```

Comment:
You should consider grabbing the second parameter returned from a type assertion to validate it worked.